pkg/logs/client/http/destination.go

```diff
@@ -223,15 +223,17 @@ func (d *Destination) sendAndRetry(payload *message.Payload, output chan *messag
 
 		d.retryLock.Lock()
 		backoffDuration := d.backoff.GetBackoffDuration(d.nbErrors)
+		d.retryLock.Unlock()
 		d.blockedUntil = time.Now().Add(backoffDuration)
```

Comment:
This is now a data race.

Reply:
Could you expand on that ? How does the behavior change ?

Reply:
I mean the assignment to `blockedUntil`: two threads writing to the same location without synchronization, a definition for a write-write data race. We may end up with weird value inside the `blockedUntil`. Also later, in `waitForBackoff` we read it without synchronization too, which is a read-write race. We may read a value from `blockedUntil` that we didn't write.

Reply:
Indeed, sorry for missing that. I suppose we could make `blockedUntil` a local variable then which I did in the last commit. Do you have any other suggestion ?